profileservice/api/controllers/profiles.py

```diff
@@ -712,6 +714,7 @@ def pii_put(pid=None):
         logging.error("PII PUT " + json.dumps(msg_json))
```

Comment:
@ywkim312,  In case of an exception, for PII, `msg_json` is getting printed. Please make sure that only the newly created `log_json` which has minimal information gets printed even in such cases.

Reply:
You can't print out the info in log_json because there is no information about the pid and uuid. So just printing out the error message with the request pid that doesn't exist. Other error printing outs are the same. It can't create the information about the pid or uuid, just error message with requested pid.